app/elastic/queries/person.py

```diff
@@ -1,5 +1,28 @@
 from elasticsearch_dsl import query
 
+# List of French stop words (from Elasticsearch's _french_ list)
+STOP_WORDS = {
+    "de",
+    "la",
+    "le",
+    "du",
+    "des",
+    "et",
+    "à",
+    "au",
+    "aux",
+    "les",
+    "un",
+    "une",
+    "dans",
+    "par",
+}
+
+
+def remove_stop_words(text):
+    # Split the text and filter out any stop words
+    return " ".join([word for word in text.split() if word.lower() not in STOP_WORDS])
```

Comment:
Nice and concise. I wonder if this is performant and can be in anyway a bottleneck (I really doubt that it is)

Reply:
In theory, the function could become a bottleneck, but in practice, it likely won't be an issue since you're only dealing with short inputs—around 5 to 6 words at most. Unless users consistently input very long texts into the filter, the performance impact should be minimal.